extensions/reactive-db2-client/deployment/src/main/java/io/quarkus/reactive/db2/client/deployment/ReactiveDB2ClientProcessor.java

```diff
@@ -71,6 +73,9 @@ ServiceStartBuildItem build(BuildProducer<FeatureBuildItem> feature,
         boolean isDefault = true; // assume always the default pool for now
         vertxPool.produce(new VertxPoolBuildItem(db2PoolValue, DatabaseKind.DB2, isDefault));
 
+        // Enable SSL support by default
+        sslNativeSupport.produce(new ExtensionSslNativeSupportBuildItem(Feature.REACTIVE_DB2_CLIENT));
```

Comment:
I wonder if `DataSourceReactiveDB2Config` shouldn't be a build-time config. Then this would only be produced only if `ssl` is enabled